### PR TITLE
governance script to enable bn254

### PR DESCRIPTION
**Pull request by zjma**

### CODE REVIEW

File: metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable BN254 algebra",
+  "description": "Enable BN254 algebra",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core"
```
Comment (sherry-x) [inline metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json:5]: Put the real url to the AIP discussion instead

File: metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable BN254 algebra",
+  "description": "Enable BN254 algebra",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```
Comment (sherry-x) [inline metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json:4]: Source url is the .move file in this repo

File: metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json (hunk 1 above)
Comment (zjma) [reply metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json]: fixed

File: metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json (hunk 2 above)
Comment (zjma) [reply metadata/2024-07-17-enable-bn254/enable_bn254_algebra.json]: fixed

**Review by sherry-x — APPROVED** [review]

**Pull request merged**